flink-yarn-tests/src/test/java/org/apache/flink/yarn/YarnTestBase.java

```diff
@@ -473,11 +472,8 @@ private static void writeHDFSSiteConfigXML(Configuration coreSite, File targetFo
     public static void ensureNoProhibitedStringInLogFiles(
             final String[] prohibited, final Pattern[] whitelisted) {
         File cwd = new File("target/" + YARN_CONFIGURATION.get(TEST_CLUSTER_NAME_KEY));
-        Assert.assertTrue(
-                "Expecting directory " + cwd.getAbsolutePath() + " to exist", cwd.exists());
-        Assert.assertTrue(
-                "Expecting directory " + cwd.getAbsolutePath() + " to be a directory",
-                cwd.isDirectory());
+        assertThat(cwd).exists();
+        assertThat(cwd).isDirectory();
```

Comment:
Oh wow there actually are file-specific assertions. We could use these also in the other tests bit more, no?

Reply:
@zentol Yes~. Thank you for recommending me the  assertJ automatic migration tool last time. Happily, I also found an idea IDE plugin called Assertions2AssertJ. The plugin could remind developers of code-lines need to migrate by marking highlights, which would be  a good supplement to automation tools.